### DISCUSSION TITLE
프로그래머스 Lv.2 풀이 (Feat. Javascript) | GIVEN's LOG

### DISCUSSION THREAD

**Discussion by giscus** [osanThor/blog#7] [Comments]
# 프로그래머스 Lv.2 풀이 (Feat. Javascript) | GIVEN's LOG

프로그래머스 Lv.2 정답률 6~70대 문제 Javascript 풀이

https://blog.given-log.com/post/dev/20241124-programers-lv2

**Comment by Hyuunw00** [comment]
블로그 완전 깔끔하고 멋져요~!

**Reply by osanThor** [reply]
감사합니다 ㅎㅎ
